README.md

```diff
@@ -9,6 +9,66 @@ Blox is being delivered as a managed service via the Amazon ECS Console, API and
 Blox schedulers are built using AWS primitives, and the Blox designs and code are open source. If you are interested in learning more or collaborating on the designs, please read the [design](docs/daemon_design.md).
 If you are currently using Blox v0.3, please read the [FAQ](docs/faq.md).
 
+### Project structure
+For an overview of the components of Blox, run:
+
+```
+./gradlew projects
+```
+
+### Testing
+To run the full unit test suite, run:
+
+```
+./gradlew check
```

Comment:
That's the Gradle idiom, and the default task that is provided by the Java plugin: https://docs.gradle.org/3.3/userguide/java_plugin.html#sec:java_tasks

I figured it was easier to exclude the end-to-end tests from the default task run by CI (see the `build.gradle` in the `end-to-end-tests` project), instead of reconfiguring every single unit test task to have a different name.